bundle/manifests/3scale-operator.clusterserviceversion.yaml

```diff
@@ -180,6 +180,10 @@ metadata:
     repository: https://github.com/3scale/3scale-operator
     support: Red Hat
     tectonic-visibility: ocs
+  labels:
```

Comment:
changes needs to be done in the `config/manifests/bases/3scale-operator.clusterserviceversion.yaml`

Reply:
this is done, resolving this conversation.